app/controllers/v0/veteran_readiness_employment_claims_controller.rb

```diff
@@ -33,5 +33,20 @@ def filtered_params
     def short_name
       'veteran_readiness_employment_claim'
     end
+
+    def encrypted_user(user)
+      user_struct = OpenStruct.new(
+        participant_id: user.participant_id,
+        pid: user.participant_id,
+        edipi: user.edipi,
+        vet360ID: user.vet360_id,
+        dob: user.birth_date,
+        ssn: user.ssn,
+        loa3: user.loa3?,
+        uuid: user.uuid,
+        va_profile_email: user&.va_profile_email
+      )
+      KmsEncrypted::Box.new.encrypt(user_struct.to_h.to_json)
+    end
```

Comment:
Also consider just using current_user directly instead of passing it in 

```
def encrypted_user
  user_struct = OpenStruct.new(
    participant_id: current_user.participant_id,
    pid: current_user.participant_id,
    edipi: current_user.edipi,
    vet360ID: current_user.vet360_id,
    dob: current_user.birth_date,
    ssn: current_user.ssn,
    loa3: current_user.loa3?,
    uuid: current_user.uuid,
    va_profile_email: current_user.va_profile_email
  )
  KmsEncrypted::Box.new.encrypt(user_struct.to_h.to_json)
end
```

Reply:
dang I should have caught that, my bad!